pkg/standalone/standalone.go

```diff
@@ -429,6 +434,32 @@ func runPlacementService(wg *sync.WaitGroup, errorChan chan<- error, dir, versio
 	errorChan <- nil
 }
 
+func moveDashboardFiles(extractedFilePath string, dir string, errorChan chan<- error) (string, error) {
```

Comment:
no need to pass errorChan in the method. Just return ( string, error ) from this method.